clusters/sandbox-values.yaml

```diff
@@ -7,6 +7,8 @@ global:
 httpsEgressSafelist:
 - name: sqs-eu-west-2
   fqdn: sqs.eu-west-2.amazonaws.com
+- name: queue-eu-west-2
+  fqdn: eu-west-2.queue.amazonaws.com # legacy endpoint
```

Comment:
dirty comment is dirty. 

maybe put info in the git commit message instead for future travelers?